test/conditionals_test.dart

```diff
@@ -159,6 +161,43 @@ class SequentialModule extends Module {
   }
 }
 
+class SignalRedrivenSequentialModule extends Module {
```

Comment:
This module could be a lot simpler for just testing redriven things, don't need so many irrelevant signals.

Reply:
Oh ya, thanks for the reminder! 👍